tests/threads/tjsthreads.nim

```diff
@@ -0,0 +1,5 @@
+discard """
+  targets: "js"
+"""
+
+echo 123
```

Comment:
I think this works "by accident" and could break in future because there is no corresponding `output: "123"` in spec, so might break megatest once we implement megatest-per-unique-option.
in any case i'd prefer:
```nim
discard """
  targets: "c cpp js"
  matrix: "--threads"
"""
discard
```